zcncore/transactionauth_base.go

```diff
@@ -13,34 +13,40 @@ import (
 )
 
 type TransactionWithAuth struct {
-	*Transaction
+	t *Transaction
```

Comment:
not sure I understand correctly, how exposing the embedded field can fix anything?

Reply:
no, just reverted changes. the issue is some method should be overloaded, some not. but they are confused in the #263 . It makes  indefinite loop. eg submitTxn.
It is difficult to test all features. Because the `TransactionWithAuth` feature is used blot+gosdk+zAuth, we have to verify all features. it is difficult for me now. so just revert changes to unblock mobile team first. You can assign the refactor to someone, and verify all changes on bolt+gosdk+zauth.

Reply:
Ah, got it thanks